microsoftgraph/microsoft-graph-toolkit
# [accessibility] storybook

#1103 
#1087 
#1085 

 

### PR Type


Bugfix







### Description of the changes
Fixes storybook's accessibility issues

### PR checklist


### Other information


### How to test
* Observe that focus order starts with the sidebar and not footer
* Observe that iframe has a title property
* Observe that the page has a level 1 heading
